internal/controller/components/llamastackoperator/llamastackoperator_support.go

```diff
@@ -0,0 +1,40 @@
+package llamastackoperator
+
+import (
+	"github.com/opendatahub-io/opendatahub-operator/v2/api/common"
+	componentApi "github.com/opendatahub-io/opendatahub-operator/v2/api/components/v1alpha1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/internal/controller/status"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
+	odhdeploy "github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+)
+
+const (
+	ComponentName = componentApi.LlamaStackOperatorComponentName
+
+	ReadyConditionType = componentApi.LlamaStackOperatorKind + status.ReadySuffix
+)
+
+var (
+	ManifestsSourcePath = map[common.Platform]string{
+		cluster.SelfManagedRhoai: "overlays/rhoai",
+		cluster.ManagedRhoai:     "overlays/rhoai",
+		cluster.OpenDataHub:      "overlays/odh",
+	}
```

Comment:
it does not have such path, right?

Reply:
I'm waiting on a [PR](https://github.com/opendatahub-io/llama-stack-k8s-operator/pull/5) to merge that will add them.